tests/page/page-aria-snapshot-ai.spec.ts

```diff
@@ -231,3 +231,25 @@ it('should gracefully fallback when child frame cant be captured', async ({ page
       - iframe [ref=e3]
   `);
 });
+
+it('should include active element information', async ({ page }) => {
+  await page.setContent(`
+    <button id="btn1">Button 1</button>
+    <button id="btn2" autofocus>Button 2</button>
```

Comment:
Please add another test with a couple of input fields, and then focus one of them using `locator.focus()` call.